spec/support/features/idv_helper.rb

```diff
@@ -30,7 +30,7 @@ def fill_out_phone_form_fail
   end
 
   def click_idv_continue
-    click_on t('forms.buttons.continue')
+    click_on t('forms.buttons.continue'), match: :first
```

Comment:
should we add a `data-` or class tag to make sure we find the right one?

Reply:
> should we add a `data-` or class tag to make sure we find the right one?

It's a bit tricky since the helper is used on many (all?) steps of the flow, not just the document capture step. So if we were to add an attribute especially for the tests, it would need to be on every step's form.

I'd thought about a few alternatives here, also lightly touched on in the extended commit message of 1e65ad7ef34f3895918e9a84f526e29399772a28, revolving around the fact that only one button will actually be visible to the user at any given time. But at least from what I can tell by using the default driver, it's not actually easy to determine which is visible, without hard-coding selectors like detecting a `.hidden` ancestor (to be replaced in BassCSS migration).

We can maybe rely on the fact that the ["normal" submit button is a `button` element](https://github.com/18F/identity-idp/blob/580ab93403535311de88b8b5236dede8fd6e1ceb/app/views/idv/doc_auth/_submit_with_spinner.html.erb#L1-L3), whereas the [hidden-by-default button is an `input` element](https://github.com/18F/identity-idp/blob/580ab93403535311de88b8b5236dede8fd6e1ceb/app/views/idv/_acuant_sdk_document_capture_form.html.erb#L44-L52).

A few thoughts occur to me:

- I don't think it really matters which one we click on, to the point about finding the "right" one
- I'm wondering now why we're rendering two in the first place 🤔 , and if we could eliminate one

Reply:
> * I'm wondering now why we're rendering two in the first place 🤔 , and if we could eliminate one

From what I can tell, "Submit" is only supposed to be shown on the existing capture steps if either of the following conditions applies:

- JavaScript is disabled (fallback form)
- JavaScript is enabled, and a successful capture has been completed

So in the case JS is enabled and capture hasn't been completed yet, there's no submit button present.

The current JavaScript is set up to handle these as distinct buttons. It could be consolidated, but it's not especially straight-forward. And it eventually becomes redundant if/when we eliminate the current flow.

Reply:
tl;dr I'd lean toward keeping this as-is. We could switch this back to only allowing a single matched element once we're eliminating the pre-React flow?

Reply:
> tl;dr I'd lean toward keeping this as-is. We could switch this back to only allowing a single matched element once we're eliminating the pre-React flow?

👍 that seems good to me! It jumped out as weird but given the info above definitely seems like the simplest choice